test/test_functional_tensor.py

```diff
@@ -163,20 +168,24 @@ def test_adjustments(self):
                 # difference in values caused by (at most 5) truncations.
                 max_diff = (ft_img - f_img).abs().max()
                 max_diff_scripted = (sft_img - f_img).abs().max()
-                self.assertLess(max_diff, 5 / 255 + 1e-5)
-                self.assertLess(max_diff_scripted, 5 / 255 + 1e-5)
+                self.assertLess(max_diff, 5 + 1e-5)  # TODO: 5 / 255, not 5
+                self.assertLess(max_diff_scripted, 5 + 1e-5)  # TODO: idem
                 self.assertTrue(torch.equal(img, img_clone))
 
             # test for class interface
-            f = transforms.ColorJitter(brightness=factor.item())
+            f = transforms.ColorJitter(brightness=bcs_factor)
+            scripted_fn = torch.jit.script(f)
+            scripted_fn(img)
+
+            f = transforms.ColorJitter(contrast=bcs_factor)
             scripted_fn = torch.jit.script(f)
             scripted_fn(img)
 
-            f = transforms.ColorJitter(contrast=factor.item())
+            f = transforms.ColorJitter(saturation=bcs_factor)
             scripted_fn = torch.jit.script(f)
             scripted_fn(img)
 
-            f = transforms.ColorJitter(saturation=factor.item())
+            f = transforms.ColorJitter(hue=abs(hue_factor))
```

Comment:
Why do you use abs here ?

Reply:
Because the `ColorJitter` class only takes `hue` inputs that are non negative: https://github.com/pytorch/vision/blob/c547e5cbbf25e46870abb2e925b42c67c3367afc/torchvision/transforms/transforms.py#L1024

Reply:
I also had to create 2 new variables, `bcs_factor` and `hue_factor` because after the inner loop, once `factor` has been overwritten, I still needed the original `factor` and `hue_factor`